core/condition.go

```diff
@@ -397,18 +397,17 @@ func blockNum(result *Result, first, second string) bool {
 		return false
 
 	}
-
-	result.AddError(fmt.Sprintf("rpc: %d, reference: %d", firstI, secondI))
-	return !(absInt(secondI-firstI) > deltaThreshold)
+	result.AddError(fmt.Sprintf("rpc: %d, reference: %d, diff: %d", firstI, secondI, secondI-firstI))
+	return !(secondI-firstI > deltaThreshold)
 }
 
-func absInt(number int64) int64 {
-	if number < 0 {
-		return -number
-	} else {
-		return number
-	}
-}
+// func absInt(number int64) int64 {
```

Comment:
If we are not using this function we can delete it not comment it

Reply:
Please un-comment this function, it is required to not have false alarms.